tests/testthat/_snaps/update-news.md

```diff
@@ -56,14 +71,17 @@
 
 # Can parse PR merge commits - internet error
 
-    [
-      {
-        "description": "PLACEHOLDER https://github.com/cynkra/fledge/pull/332 (#332).",
-        "type": "Uncategorized",
-        "breaking": false,
-        "scope": "NA"
-      }
-    ] 
+    Code
+      extract_newsworthy_items(
+        "Merge pull request #332 from cynkra/conventional-parsing")
+    Message
+      ! Could not get title for PR #332 (no internet connection)
+    Output
+      # A tibble: 1 x 4
+        description                                                type  break~1 scope
+        <chr>                                                      <chr> <lgl>   <lgl>
+      1 PLACEHOLDER https://github.com/cynkra/fledge/pull/332 (#3~ Unca~ FALSE   NA   
```

Comment:
Why are we moving away from JSON here?

Reply:
Because if we used JSON here we would not snapshot the message "Could not get title for PR #332 (no internet connection)". 

Should I tweak `expect_snapshot_tibble()` further so it'd snapshot everything?

Reply:
If that message is an error, we don't need to care about this just yet?

Reply:
Ok